11ty/src/_includes/layouts/base.njk

```diff
@@ -7,10 +7,25 @@
     <title>
       {{ title }} | {{ meta.siteName }}
     </title>
+    <!-- Open Graph -->
+    <meta property="og:title" content="{{ title }}"/>
+    <meta property="og:type" content="website"/>
+    <meta property="og:description" content="{{ meta.socialDescription }}"/>
+    <meta property="og:image" content="{{ '/img/social-card.png' | url }}"/>
+    <meta property="og:url" content="{{ meta.url }}"/>
+    <meta
+    property="og:locale" content="en_US"> <!-- Twitter Card -->
+    <meta name="twitter:card" content="summary_large_image"/>
+    <meta property="twitter:url" content="{{ meta.url }}">
```

Comment:
Will this eventually reflect the current page URL, using `meta.url` as default?

Reply:
Yes. We can set up info in the page frontmatter, too.